pkg/users/users.js

```diff
@@ -153,15 +161,33 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
+async function getLastlogPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        return "/var/run/utmp";
+    } catch (err1) {
+        try {
+            await cockpit.spawn(["test", "-e", "/var/lib/lastlog/lastlog2.db"], { err: "ignore" });
+            return "/var/lib/lastlog/lastlog2.db";
```

Comment:
Some tests do this:

```
m.execute("truncate -s0 /var/log/{[bw]tmp,lastlog} /var/run/utmp")
```

Should they also truncate lastlog2.db?

Reply:
lastlog2.db is a sqlite database. truncating it might have unintended consequences(if it exists) so no

Reply:
Alright, we can worry about this when enabling testing on opensuse by default.